docs/installation.md

```diff
@@ -94,23 +74,86 @@ read -r -d '' TOPOLOGY <<'EOF'
         "node_egress_nic": "eth0",
         "node_mesh_ip": "10.0.1.1",
         "node_mesh_nic": "eth1",
-        "etcd_master": true
+        "etcd_master": true,
+        "primary_node": true,
+        "network_node": true,
+        "hypervisor": true,
```

Comment:
Has this JSON been cut off?

Reply:
No, that's me failing to operate a computer during a cut and paste.